testpilot/frontend/static-src/styles/modules/_experiment-list.scss

```diff
@@ -185,3 +178,13 @@
   height: $grid-unit * 4.8;
   opacity: .9;
 }
+
+[data-hook='experiment-page'] {
```

Comment:
@chuckharmston I knocked the need for this out in my PR into your fork...didn't see your override here, you can kill it entirely :)